bbc/simorgh
# Fix Next.js local standalone build

Overall changes
======
- Fixes the Next.js local standalone build by copying across the static assets into the `standalone` folder. Not doing this will still build the app, but the static assets will 404, resulting in unexpected behaviour when running a compiled build locally.
- [This page](https://nextjs.org/docs/pages/api-reference/next-config-js/output#automatically-copying-traced-files) explains how `standalone` builds don't automatically copy the statically generated files, so we need to do this manually if we  want to run a compiled version of the Next.js app locally
- This only affects running the app locally when built using the `build` or `build:local` commands. This doesn't impact Test/Live deployments

Testing
======
1. Navigate to the `ws-nextjs-app` directory
2. Run `yarn build` or `yarn build:local`
3. Once the build completes, run `yarn start`
4. Navigate to http://localhost:7081/pidgin/live/c7p765ynk9qt
5. Open the developer console in the browser and confirm that there isn't a large list of 404s (note, there will likely be a 404 and/or error for the `manifest.json` file, this is fine for now)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
